### ISSUE TITLE
[suggestion] cohere-asr-ja should replaced over cohere-asr-ja-v0.1

### ISSUE THREAD

**Issue by rikimtasu** [CrispStrobe/CrispASR#127]
Just checked,it was released 2 days ago (24/05/2026) while cohere-asr-ja-v0.1 was released 23 days ago
It should replace it for overall CER improvement.
https://huggingface.co/CKHO/cohere-asr-ja-GGUF

**Comment by CKwasd** [comment]
I did a local test for the Japanese Cohere ASR GGUF files.

Model repo:

https://huggingface.co/CKHO/cohere-asr-ja-GGUF

Test setup:

- CrispASR 0.6.10
- Vulkan backend
- RTX 3080
- Main model tested: `cohere-asr-ja-q6_k.gguf`
- Test audio:
  - simple Japanese dialogue
  - Japanese cover song / singing
  - long ASMR-style quiet speech
- Compared no-VAD, FireRed VAD, Whisper VAD, and several FireRed VAD settings.
- Reference SRTs were used for rough normalized CER comparison, plus manual transcript review.

## Quantization rating

| Quant | Size | Quality | Observed speed | Suggested role |
|---|---:|---:|---:|---|
| F16 | 4.14 GB | 5/5 | 20.91x | Reference quality, fastest in this local test |
| Q8_0 | 2.42 GB | 5/5 | 15.41x | Near-F16 quality, smaller |
| Q6_K | 1.98 GB | 4.5/5 | 14.53x | Best default balance |
| Q5_0 | 1.74 GB | 4/5 | 13.79x | Smaller practical option |
| Q4_K | 1.51 GB | 3/5 | 16.70x | Smallest option, not always most accurate |

## VAD notes

VAD is not always better. In my Q6_K tests, clean dialogue was best without VAD, while song/music improved a lot with FireRed VAD.

| Audio | no-VAD CER | FireRed VAD CER | Whisper VAD CER | Best observed |
|---|---:|---:|---:|---|
| simple dialogue | 0.132 | 0.176 | 0.192 | no-VAD |
| cover song | 0.223 | 0.112 | 0.136 | FireRed VAD |
| ASMR / quiet long audio | 0.164 | 0.172 | 0.173 | no-VAD by CER, VAD better for segmentation |

One edge case I found seems specific to the newer model under some VAD segmentation/threshold conditions. The slow, drawn-out phrase `な、の、で、す、が` was handled correctly by the old v0.1 Q4_K with FireRed VAD, and also by the new model without VAD. However, the new Q6_K with one FireRed VAD setting decoded it as repeated filler-like text.

This may indicate that the newer fine-tuned model is less robust to some VAD-segmented edge cases, especially slow drawn-out syllables. I would not call it overfitting without more tests, because the same phrase is handled correctly by the new model without VAD. Still, it seems worth documenting that users may need to tune or disable VAD depending on the audio type.

## Suggested default

```text
cohere-asr-ja-q6_k.gguf
```

Suggested usage:

- Normal Japanese dialogue: Q6_K without VAD
- Song / music / long quiet speech: Q6_K with FireRed VAD
- If VAD over-splits clean speech: try no-VAD or adjust VAD settings

Overall, I think this repo is a good candidate for the recommended Japanese Cohere ASR GGUF:

https://huggingface.co/CKHO/cohere-asr-ja-GGUF

**Comment by CrispStrobe** [comment]
Added `CKHO/cohere-asr-ja-GGUF` as the default Japanese cohere variant in commit cf6e1f53. The older v0.1 entries are kept for backwards compatibility.

Usage:
```bash
crispasr --backend cohere -m cohere-asr-ja-q4_k.gguf --auto-download -l ja -f audio.wav
```

Or with `--hf-repo`:
```bash
crispasr --hf-repo CKHO/cohere-asr-ja-GGUF:cohere-asr-ja-q4_k.gguf --backend cohere -l ja -f audio.wav
```

**Comment by rikimtasu** [comment]
Thanks, just saw this get added, issue can be closed for now
